tests/e2e-playwright/tests/conftest.py

```diff
@@ -390,15 +407,54 @@ def _(
             f"Open project in {product_url=} as {product_billable=}",
         ) as ctx:
             waiter = SocketIOProjectStateUpdatedWaiter(expected_states=expected_states)
+            timeout = 60000 if template_id else 180000
             with (
-                log_in_and_out.expect_event("framereceived", waiter),
+                log_in_and_out.expect_event("framereceived", waiter, timeout=timeout),
```

Comment:
```suggestion
                log_in_and_out.expect_event("framereceived", waiter, timeout=timeout + 10 * SECOND),
```
otherwise both context will error at the same time and it will be difficult to find which one failed